RecoParticleFlow/PFClusterTools/src/PFEnergyCalibration.cc

```diff
@@ -457,10 +530,18 @@ std::ostream& operator<<(std::ostream& out,
     functType["PFfb_ENDCAP"] = PerformanceResult::PFfb_ENDCAP;
     functType["PFfc_BARREL"] = PerformanceResult::PFfc_BARREL;
     functType["PFfc_ENDCAP"] = PerformanceResult::PFfc_ENDCAP;
-    functType["PFfaEta_BARREL"] = PerformanceResult::PFfaEta_BARREL;
-    functType["PFfaEta_ENDCAP"] = PerformanceResult::PFfaEta_ENDCAP;
-    functType["PFfbEta_BARREL"] = PerformanceResult::PFfbEta_BARREL;
-    functType["PFfbEta_ENDCAP"] = PerformanceResult::PFfbEta_ENDCAP;
+    //functType["PFfaEta_BARREL"] = PerformanceResult::PFfaEta_BARREL;
+    //functType["PFfaEta_ENDCAP"] = PerformanceResult::PFfaEta_ENDCAP;
+    //functType["PFfbEta_BARREL"] = PerformanceResult::PFfbEta_BARREL;
+    //functType["PFfbEta_ENDCAP"] = PerformanceResult::PFfbEta_ENDCAP;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant